.github/workflows/lint.yml

```diff
@@ -24,6 +24,7 @@ jobs:
                   pip install vapoursynth-portable==${{ matrix.versions }}
                   pip install -r requirements.txt
                   pip install -r requirements-dev.txt
+              id: dependencies
```

Comment:
The mypy step below wants to reference this step but this was missing an ID, so the mypy step was never running. I don't really think the `if` is needed anyways because if this dependencies step fails we won't make it to the mypy step anyways.